src/components/helpers/AuthenticatorHelper.ts

```diff
@@ -0,0 +1,41 @@
+import OneWelcomeSdk, {Types} from 'onewelcome-react-native-sdk';
+
+const getAllAuthenticators = async (
+  setRegisteredAuthenticators: (authenticators: Types.Authenticator[]) => void,
+  setAllAuthenticators: (authenticators: Types.Authenticator[]) => void,
+  setPreferred: (authenticator: Types.Authenticator) => void,
```

Comment:
would change setPreferred to setPreferredAuthenticator like the two other params; but unsure if this would interfere with the const below that has the same name